components/style/properties/longhand/font.mako.rs

```diff
@@ -954,47 +954,104 @@ ${helpers.single_keyword("font-variant-position",
     }
 </%helpers:longhand>
 
-// https://www.w3.org/TR/css-fonts-3/#propdef-font-language-override
-<%helpers:longhand name="font-language-override" products="none" animatable="False" extra_prefixes="moz"
+<%helpers:longhand name="font-language-override" products="gecko" animatable="False" extra_prefixes="moz"
                    spec="https://drafts.csswg.org/css-fonts-3/#propdef-font-language-override">
+    use std::fmt;
+    use style_traits::ToCss;
+    use byteorder::{BigEndian, ByteOrder};
     use values::HasViewportPercentage;
-    use values::computed::ComputedValueAsSpecified;
-    pub use self::computed_value::T as SpecifiedValue;
-
-    impl ComputedValueAsSpecified for SpecifiedValue {}
     no_viewport_percentage!(SpecifiedValue);
 
+    #[derive(Debug, Clone, PartialEq, Eq)]
+    #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+    pub enum SpecifiedValue {
+        Normal,
+        Override(String),
+    }
+
+    impl ToCss for SpecifiedValue {
+        fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+            use cssparser;
+            match *self {
+                SpecifiedValue::Normal => dest.write_str("normal"),
+                SpecifiedValue::Override(ref lang) =>
+                    cssparser::serialize_string(lang, dest),
+            }
+        }
+    }
+
     pub mod computed_value {
-        use std::fmt;
+        use std::{fmt, str};
         use style_traits::ToCss;
+        use byteorder::{BigEndian, ByteOrder};
+        use cssparser;
 
         impl ToCss for T {
             fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-                match *self {
-                    T::Normal => dest.write_str("normal"),
-                    T::Override(ref lang) => write!(dest, "\"{}\"", lang),
+                if self.0 == 0 {
+                    return dest.write_str("normal")
                 }
+                let mut buf = [0; 4];
+                BigEndian::write_u32(&mut buf, self.0);
+                let slice = if cfg!(debug_assertions) {
+                    str::from_utf8(&buf).unwrap()
+                } else {
+                    unsafe { str::from_utf8_unchecked(&buf) }
```

Comment:
Let's add a comment here with something like "Safe because we ensure it's ASCII during parsing", or something like that.